protocols/cs2d.js

```diff
@@ -4,47 +4,41 @@ class Cs2d extends Core {
     async run(state) {
         {
             const reader = await this.sendQuery(
-                Buffer.from('\x01\x00\x03\x10\x21\xFB\x01\x75\x00', 'binary'),
-                Buffer.from('\x01\x00\xfb\x01', 'binary')
+                Buffer.from('\x01\x00\xFB\x01\xF5\x03\xFB\x05', 'binary'),
+                Buffer.from('\x01\x00\xFB\x01', 'binary')
             );
-            const flags = reader.uint(1);
-            state.raw.flags = flags;
```

Comment:
Although we do check for flags, someone might still want the data, please add this back.